documenu.py

```diff
@@ -0,0 +1,109 @@
+import requests
+import os
+from dotenv import load_dotenv
+
+load_dotenv()
+
+#This endpoint enables users to search for restaurants with the purpose of returning the selected restaurant's id for endpoint2: 
+# Users have filters like zipcode, address, etc..
+# parse through to get restaurant id
+def endpoint1():
```

Comment:
The name endpoint1() does not give much information about the function's purpose. Can you please provide a descriptive name that describes what the function does.